ff/serving/coordinator/coordinator.go

```diff
@@ -352,23 +475,23 @@ func (c *Coordinator) executeJob(jobKey string) error {
 	switch job.Resource.Type {
 	case metadata.TRAINING_SET_VARIANT:
 		if err := c.runTrainingSetJob(job.Resource); err != nil {
-			return fmt.Errorf("training set job failed: %v", err)
+			statusErr := c.Metadata.SetStatus(context.Background(), job.Resource, fmt.Sprintf("%v", err))
+			return fmt.Errorf("training set job failed: %v", fmt.Sprintf("%v: %v", err, statusErr))
```

Comment:
Errorf is already an Sprintf:
			return fmt.Errorf("training set job failed: %v: %v", err, statusErr)